installers/windows/nsis/installer.nsi

```diff
@@ -81,12 +81,20 @@ Section "!Core Files" SecCore
   File /r "python-dist\python2.7\*.*"
 
   SetOutPath "$INSTDIR\lib"
-  File /r "python-dist\lib\*.*"
+  File /r /x "python-dist\lib\site-packages\oq_platform*" /x "python-dist\lib\site-packages\openquakeplatform*" "python-dist\lib\*.*"
 
   SetOutPath "$INSTDIR\lib\site-packages\openquake"
   File "src\__init__.py"
 SectionEnd
 
+Section "OpenQuake Tools" SecTools
+  SetOutPath "$INSTDIR\lib\site-packages"
+  File /r "python-dist\lib\site-packages\oq_platform*" "python-dist\lib\site-packages\openquakeplatform*"
+
+  SetOutPath "$INSTDIR\lib\site-packages\openquake\server"
+  File /oname=local_settings.py "python-dist\lib\site-packages\openquake\server\local_settings.py.standalone"
+SectionEnd
+
 Section "OpenQuake Engine demos and documentation" SecOQ
   SetOutPath "$INSTDIR"
   File "OpenQuake manual.pdf"

```

Comment:
Thi is a name of pdf? If yes should not have spaces

Reply:
Nobody will ever open it from a terminal in Windows. At the same time it's installed under `Program Files`.